### ISSUE TITLE
[Bug] Crash on opening HUD editor in main menu

### ISSUE THREAD

**Issue by Funtimes909** [MeteorDevelopment/meteor-client#2360]
### Describe the bug

The game crashes when opening the HUD editor when not in a game
11:08:35.065
net.minecraft.class_310
Render thread
Reported exception thrown!
net.minecraft.class_148: Rendering screen
	at net.minecraft.class_757.method_3192(class_757.java:890)
	at net.minecraft.class_310.method_1523(class_310.java:1143)
	at net.minecraft.class_310.method_1514(class_310.java:734)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.NullPointerException: Cannot invoke "net.minecraft.class_746.method_43696()" because "meteordevelopment.meteorclient.MeteorClient.mc.field_1724" is null
	at meteordevelopment.meteorclient.utils.misc.FakeClientPlayer.getPlayer(FakeClientPlayer.java:52)
	at meteordevelopment.meteorclient.systems.hud.modules.PlayerModelHud.render(PlayerModelHud.java:100)
	at meteordevelopment.meteorclient.systems.hud.HUD.render(HUD.java:182)
	at meteordevelopment.meteorclient.gui.screens.HudEditorScreen.method_25394(HudEditorScreen.java:264)
	at net.minecraft.class_757.method_3192(class_757.java:881)
	... 6 more
11:08:35.070
Meteor Client
Thread-17
Saving
11:08:35.695
launcher
main

11:08:35.695
monitor
Process Monitor
Process crashed with exit code -1

### Steps to reproduce

```bash
Open Meteor Client latest dev build for 1.19 and open the HUD editor
```

### Meteor Version

meteor-client-0.5.0-1485

### Minecraft Version

1.19

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by I-Got-You-Dead** [comment]
This happens with me but I believe it is caused by addons, could you send a list on your addons pls?

**Comment by Funtimes909** [comment]
I am not playing with any addons, this occurs in vanilla minecraft with nothing but meteor and fabric api

**Comment by I-Got-You-Dead** [comment]
Interesting

**Comment by ItzSkyReed** [comment]
I also have this problem without addons

**Comment by 1Jack007** [comment]
Same problem even with newest dev build

**Comment by arlomcwalter** [comment]
fixed in a0504ef
